webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingService.scala

```diff
@@ -205,8 +213,8 @@ class VolumeTracingService @Inject()(
             WKWFile.read(is) {
               case (header, buckets) =>
                 if (header.numBlocksPerCube == 1) {
-                  parseWKWFilePath(fileName.toString).map { bucketPosition: BucketPosition =>
-                    if (resolutionsMatch || bucketPosition.resolution == Point3D(1, 1, 1)) {
+                  parseWKWFilePath(fileName.toString).map { _ =>
+                    if (buckets.hasNext) {
```

Comment:
note for reviewer: buckets.hasNext should always be true here, but bug #4905 makes it not so sometimes.